CHANGELOG.rst

```diff
@@ -9,6 +9,8 @@ This document describes changes between each past release.
 **Internal changes**
 
 - Fix documentation of errors codes (fixes #766)
+- ``kinto.id_generator`` was removed from documentation since it does not
```

Comment:
Don't we want to rather fix the behavior?
I like the idea that you can configure all id_generators with one config.